client/my-sites/plugins/use-plugins/index.ts

```diff
@@ -149,12 +150,25 @@ const usePlugins = ( {
 			break;
 	}
 
+	function fetchNextPageAndStop() {
```

Comment:
nitpick, I think this one should be called `fetchNextPageOrStop`

Reply:
I went with and because its called in a recursive loop :smile: